### ISSUE TITLE
/ad ccheet Slash Command Error

### ISSUE THREAD

**Issue by amiyuy** [TorelTwiddler/AntiDaze#11]
/ad ccheet

```
Date: 2015-03-15 13:38:23
ID: 1
Error occured in: Global
Count: 2
Message: ..\AddOns\AntiDaze\AntiDaze.lua line 113:
   attempt to call global 'ADCCheet_Toggle' (a nil value)
Debug:
   AntiDaze\AntiDaze.lua:113: AD_SlashCommand()
   AntiDaze\AntiDaze.lua:46: ?()
   ..\FrameXML\ChatFrame.lua:4422:
      ..\FrameXML\ChatFrame.lua:4369
   [C]: ChatEdit_ParseText()
   ..\FrameXML\ChatFrame.lua:4076: ChatEdit_SendText()
   ..\FrameXML\ChatFrame.lua:4115: ChatEdit_OnEnterPressed()
   [string "*:OnEnterPressed"]:1:
      [string "*:OnEnterPressed"]:1
Locals:
msg = "ccheet"
_ = "Hunter"
class = "HUNTER"
(*temporary) = nil
(*temporary) = "attempt to call global 'ADCCheet_Toggle' (a nil value)"
```

**Comment by TorelTwiddler** [comment]
fixed in 6.1.0.2

**Comment by amiyuy** [comment]
Removed slash commands entirely with 6.1.0.2
